### PR TITLE
Arm extension

### PR DESCRIPTION
**Pull request by DarthCooper**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issue(s)
#25 

## Steps to Test
Use the right stick to control the arm extension. It should change the current position in shuffleboard.

## Changelog
With an extending arm we need ways to move it especially with an analog stick as requested by drive team.

## Why?
I am making this change so we can use our extending arm.

### CODE REVIEW

File: subsystems/actuators/breadboxArmController.py [hunk 1]
```diff
@@ -60,11 +48,10 @@ def isArmPositioned(self, tolerance = None):
     def getReqSubsystems(self) -> list[commands2.Subsystem]:
         return [self, self.getArmRotation()]
 
-    def setManipulator(self, angleDegrees, armLength):
+    def setManipulator(self, angleDegrees):
```
Comment (ChrisHirsch) [inline subsystems/actuators/breadboxArmController.py:51]: could use explicit typing

File: subsystems/actuators/breadboxArmController.py [hunk 2]
```diff
@@ -73,45 +60,43 @@ def setFrontBottom(self):
...
         """
-        self.setManipulator(self.setAnglesDegrees["frontBottom"], self.setArmLength["frontBottom"])
+        self.setManipulator(self.setAnglesDegrees["frontBottom"])
 
     def setFrontCenter(self):
         """
         sets the manipulator to the front center position
         """
-        self.setManipulator(self.setAnglesDegrees["frontMiddle"], self.setArmLength["frontMiddle"])
+        self.setManipulator(self.setAnglesDegrees["frontMiddle"])
```
Comment (ChrisHirsch) [inline subsystems/actuators/breadboxArmController.py:69]: consider an enumerated type instead of frontMiddle

**Review by ChrisHirsch — APPROVED** [review]
Some minor nits...doesn't necessarily need fixed now. Was tested in the field this morning.

+1

File: subsystems/actuators/breadboxArmController.py (hunk 2 above)
Comment (tuffinmuffin) [reply subsystems/actuators/breadboxArmController.py:69]: name is from dictionary from yaml. Enum conversions would be decent am.ount more work to convert to I think

File: configs/subsystems/breadboxArmWinch.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+subsystem: "actuators.breadboxWinch.Winch"
+name: "armExtension"
+description: "Arm extension for breadBox. Future may be more general arm"
+
+motor:
+  channel: 60
```
Comment (tuffinmuffin) [inline configs/subsystems/breadboxArmWinch.yml:6]: if not using why are these in cinfig? copy and paste or future improvements?

**Review by tuffinmuffin — APPROVED** [review]

**Comment by tuffinmuffin** [comment]
merging as is for competition after Saturday testing

**Pull request merged**